docker/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.4'
+
+networks:
+  backend.fame:
+
+volumes:
+  mongodb:
+  modules:
+  storage:
+  avatars:
+
+services:
+  fame_web:
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+    image: fame:latest
+    hostname: fame_web
+    container_name: fame_web
+    networks:
+      - backend.fame
+    ports:
+      - "127.0.0.1:4200:4200"
+    env_file:
+      - fame.env
+    depends_on:
+      - fame_db
+    volumes:
+      - modules:/opt/fame/fame/modules/
+      - storage:/opt/fame/fame/storage/
+      - avatars:/opt/fame/web/static/img/avatars/
+
+  fame_worker:
```

Comment:
I would expect the worker to have the modules and storage volumes since it is not a "remote" worker.

Reply:
Actually, `fame_worker` is a remote worker, not a local one. It's a separate docker container, with a dedicated IP. This container can only communicate with `fame_web` and with the db

Reply:
Alright, I missed that but it will not work.

The `updates` queue that is supported by this worker is responsible for everything related to module repository management. With current code, a "local" worker is needed to support the `updates` queue

Reply:
I fixed the problem : there is now an additional container `fame_updater` which has modules and storage volumes mounted, and behave like a local worker